Form-Controls/index.html

```diff
@@ -14,14 +14,52 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label for="name">Name</label>
+      <input type="text" id="name" name="name" required pattern="[A-Za-z\s]{2,}" title="Please enter at least 2 letters.">
+    </div>
+    <div>
+      <label for="email">Email</label>
+      <input type="email" id="email" name="email" required pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$" title="Please enter a valid email address.">
```

Comment:
Hi @Yulia-I19 ,
Any reason for using title attribute in the input element? I think placeholder attribute may be more suitable.